dkms.in

```diff
@@ -1012,7 +1012,7 @@ prepare_build()
     read_conf_or_die "$kernelver" "$arch"
 
     # Error out if build_exclude is set
-    [[ $build_exclude ]] && die 9 \
+    [[ $build_exclude ]] && die 77 \
```

Comment:
Is @xnox in the loop about this change? I believe he's a fellow maintainer in Debian/Ubuntu, correct?

I'm in favour of this change. Small request - let's have a small note in the man page. Thus anyone else relying on the old error code has a clear reference.